sources/packages/api/src-sql/EducationPrograms/Add-field-of-study-code-column.sql

```diff
@@ -0,0 +1,7 @@
+-- Add field_of_study_code to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS field_of_study_code INT NULL;

```

Comment:
Can a program be saved without a field_of_study_code?
If it is something required for the e-Cert file I would assume that it would be required here also, right?

Reply:
No, I will update as discussed.

Reply:
Updated.